error_test.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'environment.rb'
+
+class ErrorTest
+    include Environment
+    attr_accessor :board, :depot, :control, :dictionary, :poker
+  
+    def initialize
+      @board = { players: Hash.new, control: Hash.new, status: nil }
+      @control = Environment::Control.new
+    end
+  
+    def new_worker name: nil
+      begin
+        @board[:players].store name, Environment::Worker.new(name)
+      rescue NoMethodError
+        puts 'No name for worker given'
+      end
+    end
+
+    def set_worker_data name, type, data
+        begin
+            @board[:players][name].send("set_#{type}_data", data)
+        rescue NoMethodError
+            puts 'Worker or type of data not found'
+        end
+    end
+    
+    def estimate_payment_extra_hours worker, hours
+      begin
+        base_payment = @board[:players][worker].standard_shift[:payment]
+        total = base_payment * hours
```

Comment:
Remember that the last line in the method is returned by default. You could code just `base_payment * hours`